packages/server/src/orchestrator/atom.ts

```diff
@@ -7,6 +7,11 @@ interface OrchestratorAtomOptions {
 
 export const createOrchestratorAtom = (options?: OrchestratorAtomOptions) => {
   let atom = new Atom<OrchestratorState>({
+    manifestGenerator: {
+      id: '@bigtest/manifest-genertor',
+      name: '[manifest generator]',
```

Comment:
Why both an `id` and `name`?

Reply:
@jnicklas the `name` would be for logging and the id is a unique string for the service.

Neither is currently used so could definitely remove the id until a need arises.

Just spotted I spelt `generator` wrong in the `id` :)

Reply:
@jnicklas It was I that recommended using an id like `@bigtest/foo` for the services as some sort of "plugin-ish" nomenclature so that we could get a taste of what it feels like to travel a bit on that path. What if we went with something like '@bigtest/server#manifest-generator` as the id? That would allow for multiple services per package, and right now, everything is owned by the `@bigtest/server` which is an actual package.

Reply:
`@bigtest/server#manifest-generator` is definitely better, IMO.

I do kind of want to advocate for YAGNI though in that if we don't know what we are going to use these for, and we don't currently actually use them, then I don't think it's a good idea to include them at all.

Reply:
I'm definitely 👍 to that. Although `id` and `name` are already part of the interface, so we'd have to remove it elsewhere, right? Also for the name, seems like we should remove the `[]`

Reply:
I've removed the `id` until we have a clear use for it.